docs/src/modules/components/Ad.js

```diff
@@ -53,7 +53,15 @@ const inHouseAds = [
     name: 'scaffoldhub',
     link: 'https://scaffoldhub.io/?partner=1',
     img: '/static/in-house/scaffoldhub.png',
-    description: '<b>ScaffoldHub</b> - Automate building your full-stack Material-UI web-app.',
+    description: '<b>Scaffold</b><br />Automate building your full-stack Material-UI web-app.',
+  },
+  {
+    name: 'themes',
+    link:
+      'https://themes.material-ui.com/?utm_source=material_ui&utm_medium=referral&utm_campaign=in-house-2',
+    img: '/static/in-house/themes-2.jpg',
+    description:
+      '<b>Premium Templates</b><br />Sart your project with the best themes, e.g admin & dashboard.',
```

Comment:
Templates or themes? (Pick one 🙂). The "e.g." feels a bit awkward in this context.

Reply:
Haha, it's part of the A/B test 😛.

?
```suggestion
      '<b>Premium Templates</b><br />Sart your project with the best themes for admins, dashboards and more.',
```